.github/workflows/sanity-test.yaml

```diff
@@ -353,7 +354,129 @@ jobs:
           SANITY_RESTORE_SERVICE: "onedrive"
         run: |
           set -euo pipefail
-          ./sanityCheck
+          ./sanityTest
+
+##########################################################################################################################################
+
+ # Sharepoint test
```

Comment:
Just wondering if we can move the common stuff into an `.github/action` and reuse that for each of them.

Reply:
Possibly, I can look at that as a later refactor, since there's ongoing churn with the sanity tests atm.